### DISCUSSION TITLE
Nested query string

### DISCUSSION THREAD

**Discussion by MrSpock** [rwf2/Rocket#2246] [General]
Hi,
I'm evaluating using rocket on some production system but I stuck on seemingly simple case:

Queries coming to backend server via GET request are in a form:
?search[value]=james&search[mode]=regexp
I need to get "james" and "regexp" values

Query contains mixed types. Some are simple (age=20&day=November) and I'm able to extract them but those nested make my head explode :)
How to extract such nested query string ?

**Comment by zgangguoguo** [comment]
It might help you

[Revamp query string parsing and handling](https://github.com/SergioBenitez/Rocket/issues/608)#608
